software-engineering-policies/Logging/LoggingPolicy.md

```diff
@@ -0,0 +1,136 @@
+# Logging Policy
+
+This policy aims to provide guidance to software engineering teams on how the services they develop or support should be logging. Given our logging repository of choice is ElasticSearch, some of the guidance will be influenced by that service.
```

Comment:
Consider inserting linebreaks so that the width of line in the raw file is not more than 120 characters.
I find the longer lines harder to read.
Ideally would be a setting in VSCode linter for MD files.